src/org/zaproxy/zap/extension/jxbrowserlinux64/selenium/JxBrowserProvider.java

```diff
@@ -99,8 +99,8 @@ private RemoteWebDriver getRemoteWebDriver(final String proxyAddress, final int
 
     private RemoteWebDriver getRemoteWebDriverImpl(String proxyAddress, int proxyPort) {
         try {
-            if (zbf == null) {
-                zbf = new ZapBrowserFrame(false, true, false, false);
+            if (zbf == null || zbf.isClosed()) {
+                zbf = new ZapBrowserFrame(true, true, false, true);
                 chromePort = getFreePort();
             } else if (!zbf.isVisible()) {
                 zbf.setVisible(true);
```

Comment:
We should also request focus otherwise it's not noticeable that a new tab was added to the dialogue when launching another browser (I guess we only need to do that if not a browser for AJAX Spider?).